regulations/static/regulations/js/source/views/main/child-view.js

```diff
@@ -14,56 +14,26 @@ Backbone.$ = $;
 var ChildView = Backbone.View.extend({
     initialize: function(options) {
         this.options = options;
-        var cb;
 
-        this.model = this.options.model;
-        this.listenTo(MainEvents, 'section:rendered', this.setElement);
+        this.attachWayfinding();
 
-        // callback to be sent to model's get method
-        // called after ajax resolves sucessfully
-        cb = function(success, returned) {
-            if (success) {
-                if (typeof this.options.cb !== 'undefined') {
-                    this.options.cb(returned, this.options);
-                }
-
-                if (typeof this.title === 'undefined') {
-                    this.title = this.assembleTitle();
-                }
-
-                this.route(this.options);
-
-                GAEvents.trigger('section:open', this.options);
-
-                this.attachWayfinding();
-                this.render();
-            } else {
-                MainEvents.trigger('section:error');
-            }
-        }.bind(this);
-
-        // if the site wasn't loaded on this content
         if (this.options.render) {
-            this.model.get(this.options.id, cb);
-        }
-        else if (this.options.id) {
-            this.attachWayfinding();
-            MainEvents.trigger('section:sethandlers');
-            DrawerEvents.trigger('section:open', this.id);
+          if (this.title) {
+            this.title = this.assembleTitle();
+          }
+          this.route(this.options);
+          GAEvents.trigger('section:open', this.options);
+          this.render();
+        } else if (this.options.id) {
+          MainEvents.trigger('section:sethandlers');
+          DrawerEvents.trigger('section:open', this.options.id);
         }
 
         this.$sections = this.$sections || {};
-        this.activeSection = this.id;
+        this.activeSection = this.options.id;
         this.$activeSection = $('#' + this.activeSection);
 
         this.loadImages();
-        return this;
-    },
-
-    setElement: function() {
-        if (this.id) {
-            Backbone.View.prototype.setElement.call(this, '#' + this.id);
-        }
```

Comment:
I've followed along so far, but I'm not sure why this (and the associated `listenTo`) were deleted? What did they do and why is that no longer needed?